javav2/example_code/rekognition/src/main/java/com/example/rekognition/AddFacesToCollection.java

```diff
@@ -50,7 +50,6 @@ public static void main(String[] args) {
 
         String collectionId = args[0];
         String sourceImage = args[1];
-
         Region region = Region.US_EAST_2;

```

Comment:
I notice your code has a couple of different regions. Would it make sense to add a region entry to your config file?

Reply:
I will change to use same region.